instrumentation/aws-lambda/aws-lambda-events-2.2/library/src/main/java/io/opentelemetry/instrumentation/awslambdaevents/v2_2/internal/AwsLambdaEventsInstrumenterFactory.java

```diff
@@ -20,15 +21,16 @@
 public final class AwsLambdaEventsInstrumenterFactory {
 
   public static AwsLambdaFunctionInstrumenter createInstrumenter(OpenTelemetry openTelemetry) {
-    return new AwsLambdaFunctionInstrumenter(
-        openTelemetry,
+    InstrumenterBuilder<AwsLambdaRequest, Object> otelInstrumenterBuilder =
         Instrumenter.builder(
                 openTelemetry,
                 "io.opentelemetry.aws-lambda-events-2.2",
                 AwsLambdaEventsInstrumenterFactory::spanName)
             .addAttributesExtractor(new AwsLambdaFunctionAttributesExtractor())
-            .addAttributesExtractor(new ApiGatewayProxyAttributesExtractor())
-            .buildInstrumenter(SpanKindExtractor.alwaysServer()));
+            .addAttributesExtractor(new ApiGatewayProxyAttributesExtractor());
+
+    return new AwsLambdaFunctionInstrumenter(
+        openTelemetry, otelInstrumenterBuilder.buildInstrumenter(SpanKindExtractor.alwaysServer()));
```

Comment:
I think can revert these changes now (just to clean up the diff and make the POC review easier for others)